app/lib/services/voip/CallLifecycle.test.ts

```
@@ -0,0 +1,488 @@
+	describe('teardown ordering', () => {
+		it('records commands in the documented order (steps 2-4, 6)', async () => {
+			// Arrange: set up an active call in store
+			const call = makeCall({ callId: 'order-1', state: 'active' });
+			useCallStore.getState().setCall(call);
+			native.reset();
+
+			// Act
+			await callLifecycle.end('local');
+
+			// Assert: step 2 (end), step 3 (markActive ''), step 4 (markAvailable), step 6 (stopAudio)
+			const recorded = native.recorded;
```

ESLint

[error] Use object destructuring.

(prefer-destructuring)